backend/app/Http/Controllers/CategoryController.php

```diff
@@ -35,7 +40,19 @@ public function create()
      */
     public function store(Request $request)
     {
-        //
+        $request->validate([
+            'title' => ['required', 'string'],
+            'description' => ['string'],
+            'slug' => ['required', 'string', 'unique:categories,slug'],
+        ]);
+
+        $category = Category::create($request->all());
```

Comment:
```suggestion
        $category = Category::create($request->except('_token')));
```
change to `$request->except('_token')`